verifier/src/x509/certs.rs

```diff
@@ -11,28 +14,36 @@ use x509_cert::Certificate as X509Certificate;
 
 /// A certificate whose signature has not been verified.
 #[derive(Debug, PartialEq, Eq)]
-pub struct UnverifiedCertificate<'a> {
+pub struct UnverifiedCertificate {
     // In order to verify the signature, we need to access the original DER
     // bytes
-    der_bytes: &'a [u8],
-    certificate: X509Certificate,
+    der_bytes: Vec<u8>,
```

Comment:
Got rid of the lifetimes here, it prevented the PEM decoding in the code, since pem generates a vector, and since this logic inherently needs alloc behind the scenes it doesn't seem like we gain much trying to minimize vector usage.

Reply:
SGTM